http/src/test/java/com/redhat/lightblue/client/http/testing/doubles/FakeLightblueRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.lightblue.client.http.testing.doubles;
+
+import com.redhat.lightblue.client.http.HttpMethod;
+import com.redhat.lightblue.client.request.LightblueRequest;
+
+public class FakeLightblueRequest implements LightblueRequest {
```

Comment:
Why is LightblueRequest implementation called fake?

Those changes move http protocol stuff down from lightblue http client to transport. What is your recommendation for manipulating request headers? Example: I want to set Accept-Encoding header depending on lightblue configuration. This should belong to LightblueHttpClient class, because it's independent on the underlying http client implementation.